tox.ini

```diff
@@ -4,9 +4,15 @@
 # project specific local settings
 test_pattern = "test*.py"
 #
+fixture_pattern_1 = "fixtureD*.py"
```

Comment:
I think you can drop this, since we didn't end up having to copy over the other fixture file.